.github/workflows/icu4c.yml

```diff
@@ -185,6 +185,25 @@ jobs:
           cd $PREFIX/bin;
           LD_LIBRARY_PATH=../lib ./icuinfo
 
+  gcc-warnings-as-errors:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: ICU4C with gcc and treat warnings as errors.
+        env:
+          PREFIX: /tmp/icu-prefix
+          CXXFLAGS: -Wextra -Werror -Wno-return-local-addr
+          CFLAGS: -Werror
+        run: |
+          mkdir build;
+          cd build;
+          ../icu4c/source/runConfigureICU Linux/gcc --disable-layout --disable-layoutex --prefix=$PREFIX;
+          make -j -l4.5 check;
+          make -j -l4.5 install;
+          cd $PREFIX/bin;
+          LD_LIBRARY_PATH=../lib ./icuinfo
```

Comment:
What's the purpose of doing `cd` and setting `LD_LIBRARY_PATH` and then not doing anything else after that?